README.md

```diff
@@ -18,87 +18,51 @@ logo = "coralogix.gif"
 
 Here's a good example of [Coralogix becoming a listed user](https://github.com/kedacore/keda-docs/pull/182)!
 
-## Running the site locally
+## Building or serving the site locally
```

Comment:
Would be nice to keep the split between installing (and using Choco for example) and effectively running it locally

Reply:
Ok but devs don't need to install Hugo nor yarn anymore (it's all handled via NPM) so what would be left to include in the choco installation instructions?

Reply:
They might not have Node yet and you assume they have nvm